dash/include/dash/iterator/GlobIter.h

```diff
@@ -430,7 +406,7 @@ class GlobIter
    */
   constexpr bool is_local() const
   {
-    return (_myid == lpos().unit);
+    return (dash::Team::All().myid() == lpos().unit);
```

Comment:
Caution, that's not the correct team! It should be `_pattern->team().myid()` (`Team::All()` is only used in the default constructor...) 

Same problem below...